.github/dependabot.yml

```diff
@@ -0,0 +1,15 @@
+version: 2
+updates:
+  - package-ecosystem: maven
+    directory: /sms-authenticator
+    schedule:
+      interval: weekly
+    labels:
+      - "version-upgrade"
+    ignore:
+      - dependency-name: org.keycloak:keycloak-quarkus-server-app
+      - dependency-name: io.quarkus:quarkus-vertx-http-dev-ui-resources
+      - dependency-name: org.apache.maven.plugins:maven-surefire-plugin
+      - dependency-name: org.apache.maven.plugins:maven-compiler-plugin
+      - dependency-name: io.quarkus:quarkus-maven-plugin
+      - dependency-name: org.apache.maven.plugins:maven-assembly-plugin
```

Comment:
Maven assembly was replaced. I think in general it would make more sense to update the maven plugins and keycloak dependencies project wide. But that is out of scope for this PR.

So please just replace ignore by access `com.googlecode.libphonenumber:libphonenumber`.